### PR TITLE
fix(notebook): update fr and gb resources translation

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          |  MANAGER-7429
| License          | BSD 3-Clause

## Description

Update fr and gb resources translations

### :flags: Translations

- [x] TRANS-37279
- [x] TRANS-37497

### CODE REVIEW

**Comment by ganeshkumar1989** [comment]
You will have to create a new translation request to update the other languages (other than FR). You cannot directly edit the files for other languages.

**Comment by mohammed-zahaf** [comment]
@ganeshkumar1989, I already made a request for only the required languages not all.
For the GB translation, it was provided by the PU.

**Review by frenautvh — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "pci_notebook_add_resources_usecase_item_standard": "AI Notebooks standard  (ressources CPU)",
   "pci_notebook_add_resources_usecase_item_standard_description": "Moins coûteux et proposant des performances standards. Adapté à l’apprentissage, aux tests ou aux workloads nécessitant du CPU uniquement.",
   "pci_notebook_add_resources_usecase_item_advanced": "AI Notebooks avancé (ressources GPU)",
-  "pci_notebook_add_resources_usecase_item_advanced_description": "Alimenté par plus de quatre GPU et des douzaines de CPU. Propose de meilleures performances pour l’apprentissage et les workloads intensifs.",
+  "pci_notebook_add_resources_usecase_item_advanced_description": "Alimenté jusqu'à quatre GPU et des douzaines de CPU. Propose de meilleures performances pour l’apprentissage deep learning et les workloads intensifs.",
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/translations/Messages_fr_FR.json:6]:
```suggestion
  "pci_notebook_add_resources_usecase_item_advanced_description": "Alimenter jusqu'à quatre GPU et des douzaines de CPU. Propose de meilleures performances pour l’apprentissage deep learning et les workloads intensifs.",
```
no ? Seems strange past + "jusqu'à"

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/translations/Messages_fr_FR.json:6]: For me "Alimenté" seem's good

**Review by ganeshkumar1989 — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/translations/Messages_fr_FR.json (hunk 1 above)
Comment (marie-j) [reply packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/translations/Messages_fr_FR.json:6]: I definitely don't get the meaning of this sentence. The fact that it is "Alimenté" used as an adjective seems wrong here

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/translations/Messages_fr_FR.json]: I got new sentence.
Updated.

**Review by ganeshkumar1989 — APPROVED** [review]

**Review by radireddy — DISMISSED** [review]

**Comment by mohammed-zahaf** [comment]
It seems that translations are not compliant for ES and IT.
Reporter checking.

**Review by marie-j — APPROVED** [review]

**Review by cbourgois — APPROVED** [review]

**Comment by mohammed-zahaf** [comment]
Hello @antleblanc, this PR is ready to merge.
thanks.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**